modules/user-access/pages/acl-management.adoc

```diff
@@ -180,14 +180,14 @@ To grant `EXECUTE` privilege instead, replace `READ` with `EXECUTE`:
 
 [.wrap,gsql]
 ----
-GSQL > GRANT ACL PRIVILEGE READ ON example_query TO ROLE example_role
+GSQL > GRANT ACL PRIVILEGE READ ON example_query TO example_role
 ----
 
 
 === Revoke ACL privilege from a role
 The owner of a query can revoke ACL privileges on the query from a role.
-If you revoke an ACL privilege on the query from all roles, then the ACL entry status for that privilege becomes unspecified.
-When the ACL entry status for a privilege is unspecified, RBAC governs access for the corresponding operation.
+If you revoke an ACL privilege on the query from all roles, then the ACL entry status for that privilege becomes `NOBODY`.
```

Comment:
may be better to clarify that:
If using cmd like `revoke acl privilege ... from <role1>, <role2>` until the acl list becomes empty, the acl entry status becomes `Nobody`.
If using cmd `revoke acl privilege ... from all` or `revoke acl privilege ... from *`, then the acl entry becomes `Unspecified`.